paper/paper.md

```diff
@@ -0,0 +1,108 @@
+Recent developments in biochemistry, microfluidics, and microscopy have come together to bring about an “alphabet soup” of technologies that enable sampling gene expression in situ, with varying levels of spatial resolution, sensitivity, and genetic depth.
+These technologies promise to permit biologists to ask new questions about the spatial relationships between cell type and interactions between gene expression and cell morphology.
+However, these assays generate very large microscopy datasets which pose challenges for data processing using standard analysis tools in microscopy. Furthermore, many of these assays require specialized analysis to decode gene expression from multiplexed experimental designs.
+
+*starfish* is a Python library for processing images generated by microscopy-based spatial transcriptomics assays. It lets biologists build scalable pipelines that localize and quantify RNA transcripts in image data generated by any FISH or sequencing *in situ* method, from classic RNA single-molecule FISH to combinatorial barcoded assays. Image processing of an experiment is divided into fields of view (FOV) that correspond to the data produced by a microscope at a single location on a microscope slide. Built on top of scikit-image and xarray, *starfish* lets users register and pre-process images in each FOV, localize spots representing tagged RNA molecules in 3D, decode the identity of those molecules according to the experimental design, segment cells, assign the spots to cells, then aggregate spots into a cell x gene expression matrix. This spatially-annotated gene expression matrix can then be analyzed in downstream tools for single-cell biology, such as Seurat (@Seurat), Bioconductor (@Bioconductor), and ScanPy (@ScanPy).
+
+To enable large scale processing of these data, *starfish* implements on a cloud-friendly data format that supports the 5-dimensional imaging data structures that these assays generate. Furthermore, *starfish* implements comprehensive logging of all data processing steps for provenance tracking, reproducibility, and transparency. *starfish* is built on top of popular Python tools like xarray (@xarray) and scikit-image (@scikit-image).
+
+To help new users get started and support the broader single cell biology community in learning how to work with these data, *starfish* maintains example datasets and reference implementations ported from published assays, including
+MERFISH (@MERFISH),
+In Situ Sequencing (@ISS),
+osmFISH (@osmFISH),
+BaristaSeq (@BaristaSeq),
+smFISH,
+DARTFISH,
+STARmap (@starmap),
+and seqFISH (@seqFISH).
+
+*starfish* is currently in use by multiple research groups, including the Allen Institute for Brain Science, the Chan Zuckerberg Biohub, and the Zhang Lab at UC San Diego. These groups support multiple large-scale projects profiling *in situ* gene expression, including the SpaceTx consortium, the Human Cell Atlas, the BRAIN Initiative Cell Census Network, and HuBMap.
```

Comment:
Should we provide links or references to the referenced consortia? People reading the manuscript may not know who/what they are.

Reply:
That's a good idea.